### PR TITLE
Add custom license_check.yaml fo exclude corner cases

### PR DESCRIPTION
**Pull request by thiagonache**
#### Summary and Scope

Fixes license checker by adding a custom license_check.yaml file to deal with corner cases.

#### Prerequisites

- [x] I have included documentation in my PR (or it is not required)
- [x] I tested this on internal system (x) (if yes, please include results or a description of the test)

### CODE REVIEW

File: .license_check.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+add_exclude:
+  - cmd/shcd_test.go
```
Comment (jacobsalmela) [inline .license_check.yaml:25]: I think the license checker only checks edited files per PR.  May as well exclude the other tests because they all have those tags?

Comment (thiagonache) [reply .license_check.yaml:25]: yeah... good advice.

**Review by rustydb — APPROVED** [review]
